src/click-outside.directive.ts

```diff
@@ -25,11 +25,13 @@ export class ClickOutsideDirective implements OnInit, OnChanges, OnDestroy {
   @Input() excludeBeforeClick: boolean = false;
   @Input() clickOutsideEvents: string = '';
   @Input() clickOutsideEnabled: boolean = true;
+  @Input() emitOnBlur: boolean = false;
 
   @Output() clickOutside: EventEmitter<Event> = new EventEmitter<Event>();
 
   private _nodesExcluded: Array<HTMLElement> = [];
   private _events: Array<string> = ['click'];
+  private _windowBlurListener: any = this._onWindowBlur.bind(this);
```

Comment:
Nitpick: I'd prefer for this to be done in the constructor along with the other bindings.